staging/src/k8s.io/client-go/rest/watch/decoder_test.go

```diff
@@ -44,6 +44,9 @@ func getDecoder() runtime.Decoder {
 func TestDecoder(t *testing.T) {
 	table := []watch.EventType{watch.Added, watch.Deleted, watch.Modified, watch.Error, watch.Bookmark}
 
+	errs := make(chan error, 2)
```

Comment:
errs is created outside the for loop, but used inside the for loop below in multiple iterations. if it is intended to be used for just one iteration, better to move it into the for loop.

Reply:
If move it into the for loop, the errs will be created repeatedly, so i put it outside the for loop.